test/teos/unit/test_inspector.py

```diff
@@ -185,7 +183,7 @@ def test_inspect(run_bitcoind):
     appointment = inspector.inspect(appointment_data)
 
     assert (
-        type(appointment) == ExtendedAppointment
```

Comment:
I think `isinsance(appointment, Appointment)` is considered more idiomatic?

Reply:
Agreed. However, looking at this closely, it can be removed. That check comes from an older version of `Inspector.inspect` where the method had multi-return (Appointment or None), currently the method raises an exception if something goes wrong.